cl/api/templates/v2_webhooks-getting-started.html

```diff
@@ -0,0 +1,114 @@
+    <p>To create your first webhook, there are a handful of steps you will need to take:</p>
+    <ol class="space-y-2">
+      <li>Creating and receiving webhooks does not require any special access, but sooner or later you will want to have access to the RECAP API endpoints. <a class="underline" href="{% url 'contact' %}">Send us a note with your username and use case</a> to get that process started.</li>
+      <li>You need to <a class="underline" href="#receive-a-webhook">create a URL on your server</a> to receive events from CourtListener.</li>
+      <li>You need to <a class="underline" href="#setup-a-webhook">configure CourtListener to send events</a> to that URL.</li>
+    </ol>
+    <p>Once you have created the URL on your server and linked it up to ours, you can <a class="underline" href="#testing">send test events from CourtListener</a> and you can wait for events to be triggered. As you send test events or events are automatically triggered, they appear in the <a class="underline" href="{% url 'view_webhook_logs' 'logs' %}">logs section of your webhooks panel</a>.</p>
+    <p>Read on to learn more about setting up your first webhook.</p>
+  </c-layout-with-navigation.section>
+
+  <c-layout-with-navigation.section id="receive-a-webhook">
+    <h2>Receive a Webhook Event on Your Server</h2>
+    <p>Webhook events are nothing more than HTTP POST requests sent to your server. To receive these events, you will need to begin by creating a URL in your application.</p>
+    <p>Some requirements to consider for the URL that you create:</p>
+    <ul class="space-y-2">
+      <li>Our webhook system <a class="underline" href="{% url 'webhooks_docs' %}#security">does not support authentication</a>, so the URL you create should be long and random.</li>
+      <li>Our POST requests will come from one of two static IP addresses, <code>34.210.230.218</code> and <code>54.189.59.91</code>. We recommend only allowing traffic from these addresses.</li>
+      <li>Your URL must receive POST requests and should not accept any other HTTP verbs.</li>
+      <li>If you are using a web framework like Django or Ruby on Rails, the webhook URL will need to be exempted from the cross site request forgery (CSRF) protection system. If this is not done, most web frameworks will block POST requests from outside domains like ours.</li>
+      <li>To avoid <a class="underline" href="{% url 'webhooks_docs' %}#retries">timeout errors and event retries</a>, your application may need to process events asynchronously.</li>
+      <li>Your application must respond to events with a 2xx HTTP status code or else the event will be retried.</li>
+      <li>Your application should use the <a class="underline" href="{% url 'webhooks_docs' %}#headers">Idempotency-Key included in the event headers</a> to ensure that it only processes the event once.</li>
+    </ul>
+    <p>Below are examples of endpoints configured in Flask and Django</p>
+    <p><strong>Flask</strong></p>
```

Comment:
I think `<h3>` is more semantically correct here than `<p><strong>`:

```suggestion
    <h3>Flask</h3>
```